circle.yml

```diff
@@ -215,6 +215,10 @@ jobs:
     executor: linux-clang-latest
     steps:
       - checkout
+      - run:
+          name: "Run shellcheck"
+          command: |
+            git ls-files '*.sh' | xargs shellcheck --external-sources
```

Comment:
Will this fail on error?

Reply:
Yes, I checked it with removing one quote from `apply_patches.sh`.